templates/hello/simple-snippet.html

```diff
@@ -0,0 +1,209 @@
+<!-- Any instances of {{ snippet_id }} are replaced with the ID number for the
+     snippet using this template. -->
+
+<div class="snippet" id="{{ snippet_id }}-hello-simple">
+    <a href="#" class="launch-hello">
+      <img class="icon" src="{{ icon }}" />
+    </a>
+    <p>{{ text|safe }}</p>
+
+    <script type="text/javascript">
+      //<![CDATA[
+
+      (function() {
+        var snippet = document.getElementById('{{ snippet_id }}-hello-simple');
+        snippet.addEventListener('show_snippet', function() {
+          var buttons = snippet.getElementsByClassName('launch-hello');
+
+          isTargetAvailable('loop', function () {
+            sendMetric('hello-eligible-view');
+            for (var i=0; i < buttons.length; i++) {
+              buttons[i].addEventListener('click', function(e) {
+                e.preventDefault();
+                Mozilla.UITour.showMenu('loop');
+
+                // bug 1115227
+                Mozilla.UITour.registerPageID('hello-tour_OpenPanel_snippet-{{ snippet_id }}');
+
+                // bug 1113896
+                document.addEventListener('click', function(e) {
+                    Mozilla.UITour.hideMenu('loop');
+                    document.removeEventListener('click', this);
```

Comment:
Unless I'm misreading [the docs](http://mdn.beonex.com/en/DOM/element.addEventListener.html), shouldn't `this` refer DOM element the listener is attached to? `removeEventListener` requires you to pass the callback function and the same `useCapture` value as you used in `addEventListener`.